app/frontend/src/cljs/teet/ui/common.cljs

```diff
@@ -592,3 +593,19 @@
         [Paper
          (mapc (r/partial context-menu-item toggle!)
                (remove nil? items))]]]]]))
+
+(defn date-label-component
+  "Shows :label with formatted date from :value with an info icon and :tooltip"
+  [{value :value label :label tooltip :tooltip}]
+  [:div
+   {:style (merge {:color theme-colors/gray} common-styles/body-2-bold)
+    :data-cy "date-label-component"}
+   label
+   [:div
+    {:style common-styles/body-1-bold}
+    [popper-tooltip tooltip
+     [icons/action-info-outlined
```

Comment:
Not best practice to position elements with some arbitrary positional pixel values. 

if it's meant to align the icon with the text you could have the wrapping div have the `common-styles/flex-row-center`

Reply:
Just adding common-styles/flex-row-center doesn't work. the exception is thrown by align function

Reply:
Yea it requires there to be a wrapping div to which the flex-row is given.

Reply:
Fixed by wrapping into Grid container and using alignment center.